src/rpc/voting.cpp

```diff
@@ -59,9 +59,12 @@ UniValue PollToJson(const Poll& poll, const uint256 txid)
     json.pushKV("id", txid.ToString());
     json.pushKV("question", poll.m_question);
     json.pushKV("url", poll.m_url);
-    json.pushKV("sharetype", poll.WeightTypeToString());
-    json.pushKV("weight_type", (int)poll.m_weight_type.Raw());
-    json.pushKV("response_type", (int)poll.m_response_type.Raw());
+    json.pushKV("poll_type", poll.PollTypeToString());
+    json.pushKV("poll_type_id", (int)poll.m_type.Raw());
+    json.pushKV("weight_type", poll.WeightTypeToString());
+    json.pushKV("weight_type_id", (int)poll.m_weight_type.Raw());
+    json.pushKV("response_type", poll.ResponseTypeToString());
+    json.pushKV("response_type_id", (int)poll.m_response_type.Raw());
```

Comment:
I don't mind, I prefer your current approach. Just felt that since we're breaking the compatibility we should at least notify the most "use"r of the interface.

Reply:
Agreed. @startailcoon is probably the only one that really cares about this right now quite frankly. Let's let him weigh in.